Sorted sprites correctly

gadahl opened the pull request:
i did it
plz mergw

gadahl committed:
Sorted sprites correctly

i did it

Bluju reviewed (approved):
There are still a few scenarios where the render order of the sprites is incorrect but we can work around it for now.

gadahl committed:
fixes & enemy collider

Fixed the placement of some tiles so that the player now sorts properly. Added box collider and sprite sorting to enemy.

gadahl committed:
Fixed tile positions

Fixed tiles so they're all where they're supposed to be

gadahl committed:
Added enemy collider

Set the enemy up in the same way as the player. Once the sprite is changed the hitbox size will need to change as well as the sprite's pivot.

ssmith817 reviewed (approved):
Works well on my end.
